src/services.rs

```diff
@@ -158,6 +158,9 @@ pub enum ChannelKind {
     Thermostat,
     ActualTemperature,
 
+    AddRule,
```

Comment:
I'd prefer `AddThinkerbellRule`/`RemoveThinkerbellRule`. Also, doc would be useful.